src/Classes/ListingHydrator.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+
+namespace ListingsApp\Classes;
+require_once '../../vendor/autoload.php';
```

Comment:
You should be able to remove this require_once now

Reply:
removed :)